src/components/shared/Avatar/Avatar.tsx

```diff
@@ -3,6 +3,7 @@ import React, { CSSProperties, ReactNode } from 'react';
 import getIcon from './identicon';
 import Icon from '~shared/Icon';
 import styles from './Avatar.module.css';
+import { AvatarSize } from '~v5/shared/Avatar/types';
```

Comment:
Import order seems off, styles should come last as per https://github.com/JoinColony/ColonyFrontEndLivingStandard/blob/master/repo/index.md#imports